### PR TITLE
Cleaned Up CustomControls and made suggested changes and bug fixes for Project Structure/Pane

### PR DESCRIPTION
**Pull request by openbots-ff**
- Cleaned up CustomControls
- Fixed {ProjectPath} bug. Previously, {ProjectPath} wasn't being appended to the Variable List in Main when creating a new project. 
- Increased the display time for ToolTips
- Disabled unusable context menu buttons for project folder, Main, and project.config (Delete/Rename/Copy)

### CODE REVIEW

File: taskt/UI/CustomControls/CustomUIControls/UITabControl.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        //internal void DrawControl(Graphics g)
+        //{
+        //    if (!Visible)
+        //        return;
+
+        //    Brush br = new SolidBrush(Color.Black);
+        //    Brush brTab = new SolidBrush(Color.Orange);
+
+        //    Rectangle TabControlArea = ClientRectangle;
+        //    Rectangle TabArea = DisplayRectangle;
```
Comment (ob-ab) [inline taskt/UI/CustomControls/CustomUIControls/UITabControl.cs:21]: Whole class body is commented, is it intentional?

**Review by ob-ab — COMMENTED** [review]
I have left a comment to seek whether the change was intentional, please confirm.
Everything else looks good to me.

File: taskt/UI/CustomControls/CustomUIControls/UITabControl.cs (hunk 1 above)
Comment (openbots-ff) [reply taskt/UI/CustomControls/CustomUIControls/UITabControl.cs:21]: That was already commented out in the original version. I expect he was intending to make customization to the standard tab control but put it on hold

Comment (openbots-ff) [reply taskt/UI/CustomControls/CustomUIControls/UITabControl.cs:21]: I actually might use some of this code when I finish making the close buttons for tabs so I don't want to delete it yet.

Comment (ob-ab) [reply taskt/UI/CustomControls/CustomUIControls/UITabControl.cs:21]: Got it.

**Review by ob-ab — DISMISSED** [review]

**Review by CameronHerwig — DISMISSED** [review]

**Review by CameronHerwig — APPROVED** [review]

**Review by ob-ab — APPROVED** [review]

**Pull request merged**